packages/react-examples/package.json

```diff
@@ -53,7 +53,6 @@
     "@fluentui/react-hooks": "^8.2.2",
     "@fluentui/react-icons-mdl2": "^1.1.3",
     "@fluentui/react-label": "^9.0.0-alpha.9",
```

Comment:
`@fluentui/react-label` should also probably be removed from this package if it is being migrated.